core/templates/dev/head/components/alerts/AlertMessageDirective.js

```diff
@@ -33,14 +33,20 @@ oppia.directive('alertMessage', [function() {
     ],
     link: function(scope) {
       var message = scope.getMessage();
+      var timeoutVal = 1500;
```

Comment:
Can you think of a better variable name other than timeoutVal ?. (Maybe considering the `1500 ms` as general value for all other messages)

Reply:
Sure :+1: